CHANGELOG.md

```diff
@@ -6,11 +6,44 @@ All notable changes for each version of this project will be documented in this
 - `IgxCarousel`
     - **Behavioral Changes** - the `maximumIndicatorsCount` input property now defaults to `10`.
     - **Deprecation** - `CarouselIndicatorsOrientation` enum members `top` and `bottom` have been deprecated and will be removed in a future version. Use `start` and `end` instead.
+- `IgxQueryBuilderHeader`
+    - **Behavioral Changes**
+    - Legend is no longer shown.
+    - **Deprecation**
+    - The `showLegend` input property has been deprecated and will be removed in a future version. Automatic migrations are available and will be applied on `ng update`.
```

Comment:
This is still mostly relevant under the QB section rather than general, no?

Reply:
Also this can be tied with a point that I'm missing - there is a redesign of the component we might want to note.